test/sagemaker_tests/mxnet/training/integration/local/test_keras_training.py

```diff
@@ -26,6 +26,9 @@
 @pytest.mark.model("mnist")
 def test_keras_training(docker_image, sagemaker_local_session, local_instance_type,
                         framework_version, tmpdir):
+    if framework_version in ['1.9.0']:
```

Comment:
If this is meant to be PT1.9 onwards, can you use `Version` from the `packaging.version` module?

example
```
from packaging.version import Version
...
...
...
if Version(framework_version) >= Version('1.9.0'):
    ...
```

**Since this is used in a lot of places, you can also create a utility function to do this if needed**